src/main/java/com/browserup/harreader/model/HarLog.java

```diff
@@ -123,4 +129,41 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(version, creator, browser, pages, entries, comment);
     }
+
+    /**
+     * Search the entire log for the most recent entry whose request URL matches the given <code>url</code>.
+     *
+     * @param url Regular expression match of URL to find.
+     *            URLs are formatted as: scheme://host:port/path?querystring.
+     *            Port is not included in the URL if it is the standard port for the scheme.
+     *            Fragments (example.com/#fragment) should not be included in the URL.
+     *            If more than one URL found, return the most recently requested URL.
+     * @return <code>HarEntry</code> for the most recently requested URL matching the given <code>url</code> pattern.
+     */
+    public Optional<HarEntry> findEntry(Pattern url) {
```

Comment:
We should name this something to indicate it is returning the entry with the greatest starting time (vs the first, or an arbitrary one).

`findMostRecentEntry(Pattern)` perhaps?

@rkuzsma you've been naming these - do you have a preferred pattern for this?